### PR TITLE
feat: attribute and catalog refactor QA

### PR DESCRIPTION
**Pull request by micdes-pagopa**

### CODE REVIEW

File: kubernetes/configs/qa/commons.sh [hunk 1]
```diff
@@ -49,3 +56,5 @@ PAGOPA_TENANT_ID="69e2865e-65ab-4e48-a638-2037a9ee2ee7"
 
 SMTP_ADDRESS="smtp.gmail.com"
 SMTP_PORT=465
+
+AUTHORIZATION_MANAGEMENT_URL="http://interop-be-authorization-management.qa.svc.cluster.local:8088/authorization-management/0.0"
```
Comment (galales) [inline kubernetes/configs/qa/commons.sh]: in QA we use tags
```suggestion
AUTHORIZATION_MANAGEMENT_URL="http://interop-be-authorization-management.qa.svc.cluster.local:8088/authorization-management/1.0"
```

File: kubernetes/configs/qa/versions.sh
Comment (galales) [inline kubernetes/configs/qa/versions.sh:1]: in QA we used tags so far, is the use of `latest` version made on purpose?

Comment (micdes-pagopa) [reply kubernetes/configs/qa/versions.sh:1]: You're right. I'll change to the most recent prod tags for all services

Comment (beetlecrunch) [reply kubernetes/configs/qa/versions.sh:1]: It was necessary to enable the execution of tests in the previous days, but with this PR we can align them with the prod tags.

Comment (micdes-pagopa) [reply kubernetes/configs/qa/versions.sh:1]: Fixed

**Review by beetlecrunch — DISMISSED** [review]

**Pull request merged**